ocaml/utils/clflags.ml

```diff
@@ -395,7 +392,23 @@ module Extension = struct
     | "local" -> Local
     | extn -> raise (Arg.Bad(Printf.sprintf "Extension %s is not known" extn))
 
+  let disable_all_extensions = ref false             (* -disable-all-extensions *)
+
+  let disable_all () =
+    disable_all_extensions := true;
+    match !extensions with
+    | [] -> ()
+    | ls ->
+      raise (Arg.Bad(Printf.sprintf
+        "Extensions %s are incompatible with compiler flag -disable-all-extensions"
```

Comment:
(I think there is a singular vs plural issue.)

Reply:
Do you mean because `ls` is of length one?

Reply:
Yes, I mean `ls` can be of length one, and then `Extensions e`
is a bit weird; I don't think it should be blocking, though.

Reply:
Here is a better wording.

Reply:
How is that different, "extensions" is still plural.
But again, I don't think it should be blocking.

Reply:
It's different, but maybe not better :)  at least there is no `are`, seems less offensive, but I'm not a native speaker..